### PR TITLE
Changes and additions to support initial version of NBi.PowerShell ex…

### PR DESCRIPTION
**Pull request by TheAutomatingMrLynch**
…tension.

### CODE REVIEW

**Comment by TheAutomatingMrLynch** [comment]
Please let me know if I'm doing it the wrong way. :-)

File: NBi.genbiL/Action/Template/AddStringTemplateAction.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        public void Execute(GenerationState state)
+        {
+            state.Templates.Add(TemplateString);
+        }
+
+        public string Display
+        {
+            get
+            {
+                return string.Format($"Adding new Template from string '{TemplateString}'");
```
Comment (Seddryck) [inline NBi.genbiL/Action/Template/AddStringTemplateAction.cs:26]: string.Format is pointless in that case ;-)

File: NBi.Services/TestCaseManager.cs [hunk 2]
```diff
@@ -54,7 +54,16 @@ public void ReadFromQuery(string sql, string connectionString)
                 variables.Add(col.ColumnName);
         }
 
-        private DataTable content;
+        public void ReadFromDataTable(DataTable datatable)
```
Comment (Seddryck) [inline NBi.Services/TestCaseManager.cs:57]: It's true that I wrote other ```ReadFrom``` methods in the ```State``` class but they should be in the ```Action``` class. No problem, I need to refactor this and will do it.

**Review by Seddryck — APPROVED** [review]
Thx a lot for your pull request and improving NBi. 

I'll fix the two remarks that I made in a refactoring of my own code that is overdue. Small remark would have been perfact if you added tests but I'll do it in my refactoring.

**Comment by TheAutomatingMrLynch** [comment]
Thank you for creating NBi in the first place! I promise I'll add tests next time (and less silly code).

**Pull request merged**